### DISCUSSION TITLE
UH, HELP PLEASE?

### DISCUSSION THREAD

**Discussion by DelfixYT** [YoshiCrafter29/YoshiCrafterEngine#218] [Q&A]
"export/release/windows/haxe/ApplicationMain.hx:196: characters 25-37 : Too many arguments
export/release/windows/haxe/ApplicationMain.hx:143: characters 4-35 : ... Called from macro here
source/Song.hx:14: characters 19-28 : Type not found : SongEvent
source/PlayState.hx:9: characters 8-20 : Type not found : WiggleEffect
source/PlayState.hx:46: characters 8-20 : Type not found : LoadSettings
source/PlayState.hx:50: characters 25-39 : Type not found : MusicBeatState
source/lime/utils/Log.hx:9: lines 9-142 : ... Defined in this class
source/CoolUtil.hx:4: characters 8-22 : Type not found : EngineSettings
source/CoolUtil.hx:27: characters 10-18 : Type not found : Settings
source/flixel/system/frontEnds/LogFrontEnd.hx:106: characters 27-30 : Class<PlayState> has no field log
source/CoolUtil.hx:43: characters 22-27 : Type not found : Paths
source/CoolUtil.hx:59: characters 14-19 : Class<PlayState> has no field _SONG
source/CoolUtil.hx:59: characters 27-42 : Class<Song> has no field loadModFromJson
source/CoolUtil.hx:59: characters 43-52 : Type not found : Highscore
source/CoolUtil.hx:63: characters 13-18 : Class<PlayState> has no field _SONG
source/CoolUtil.hx:64: characters 13-24 : Class<PlayState> has no field isStoryMode
source/CoolUtil.hx:65: characters 13-22 : Class<PlayState> has no field startTime
source/CoolUtil.hx:66: characters 13-20 : Class<PlayState> has no field songMod
source/CoolUtil.hx:67: characters 13-25 : Class<PlayState> has no field jsonSongName
source/CoolUtil.hx:68: characters 13-28 : Class<PlayState> has no field storyDifficulty
source/CoolUtil.hx:69: characters 13-24 : Class<PlayState> has no field fromCharter
source/CoolUtil.hx:173: characters 11-16 : Type not found : Paths
source/CoolUtil.hx:174: characters 30-35 : Type not found : Paths
source/CoolUtil.hx:186: characters 11-16 : Type not found : Paths
source/CoolUtil.hx:197: characters 27-32 : Type not found : Paths
source/CoolUtil.hx:198: characters 22-27 : Type not found : Paths
source/CoolUtil.hx:206: characters 11-16 : Type not found : Paths
source/CoolUtil.hx:207: characters 30-35 : Type not found : Paths
source/CoolUtil.hx:270: characters 20-35 : Class<PlayState> has no field storyDifficulty
source/CoolUtil.hx:314: characters 12-22 : Type not found : ModSupport
source/CoolUtil.hx:317: characters 17-27 : Type not found : ModSupport
source/CoolUtil.hx:319: characters 15-32 : You can't iterate on a Dynamic value, please specify Iterator or Iterable
source/CoolUtil.hx:339: characters 15-20 : Type not found : Paths
source/CoolUtil.hx:343: characters 20-25 : Type not found : Paths
source/CoolUtil.hx:368: characters 25-43 : Class<Main> has no field supportedFileTypes
source/CoolUtil.hx:368: characters 20-43 : You can't iterate on a Dynamic value, please specify Iterator or Iterable
source/CoolUtil.hx:369: characters 30-35 : Type not found : Paths
source/CoolUtil.hx:375: characters 26-44 : Class<Main> has no field supportedFileTypes
source/CoolUtil.hx:375: characters 21-44 : You can't iterate on a Dynamic value, please specify Iterator or Iterable
source/CoolUtil.hx:376: characters 31-36 : Type not found : Paths
source/CoolUtil.hx:394: characters 25-43 : Class<Main> has no field supportedFileTypes
source/CoolUtil.hx:394: characters 20-43 : You can't iterate on a Dynamic value, please specify Iterator or Iterable
source/CoolUtil.hx:395: characters 30-35 : Type not found : Paths
source/CoolUtil.hx:401: characters 26-44 : Class<Main> has no field supportedFileTypes
source/CoolUtil.hx:401: characters 21-44 : You can't iterate on a Dynamic value, please specify Iterator or Iterable
source/CoolUtil.hx:402: characters 31-36 : Type not found : Paths
Pressione qualquer tecla para continuar. . .
Running "lime test windows -dce no "
export/release/windows/haxe/ApplicationMain.hx:196: characters 25-37 : Too many arguments
export/release/windows/haxe/ApplicationMain.hx:143: characters 4-35 : ... Called from macro here
source/Song.hx:14: characters 19-28 : Type not found : SongEvent
source/PlayState.hx:46: characters 8-20 : Type not found : LoadSettings
source/PlayState.hx:50: characters 25-39 : Type not found : MusicBeatState
source/lime/utils/Log.hx:9: lines 9-142 : ... Defined in this class
source/WiggleEffect.hx:62: characters 28-42 : Type not found : FlxFixedShader
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/utils/AssetLibrary.hx:23: lines 23-952 : ... Defined in this class
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/utils/AssetLibrary.hx:684: characters 42-58 : Void should be String
C:/HaxeToolkit/haxe/lib/lime/7,9,0/src/lime/utils/AssetLibrary.hx:684: characters 42-58 : ... For function argument 'value'
source/WiggleEffect.hx:62: characters 28-42 : Type not found : FlxFixedShader"
-build.bat
please help or give download link directly to the source folder

**Comment by KittySleeper** [comment]
build.Bat doesnt work by what ik of also what os
